advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx

```diff
@@ -0,0 +1,485 @@
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration30.png">
+</p>
+
+   We then define the “Image” column as “Image” datatype by clicking on the datatype dropdown, so we can later position it on the screen as an image.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration31.png">
+</p>
+
+   Now we connect the S_Product table as a detail to the S_Item table. To do this, we select the S_Product table at the bottom of the editor and click on the “Edit” icon. Then we click on “More” and select the S_Item as Master Table.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration32.png">
+</p>
+
+   Repeat the same for the S_Image table, because S_Image is a detail to the S_Product.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration33.png">
+</p>
+
+   Select the screen’s detail area by clicking “Details” on the orange bar at the top of the “Orders” screen. Then select the S_Image table in the lower area and position the image editor on the top right corner of the screen using drag & drop.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration34.png">
+</p>
+
+12. Open the screen in the Live Preview. With a few steps we created the Oracle Forms screen in VisionX based on the current data model. We even created an additional list view with a search function.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration35.png">
+</p>
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration36.png">
+</p>
+
+### Oracle database to EDB Postgres Advanced Server migration
+
+In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended Server using the EDBs MTK migration tool.
+
+The migration steps in connection with an Oracle Forms migration are not different from a pure Oracle to EDB Postgres Advanced Server database migration. Therefore, we do not discuss this topic in detail here.  Links to MTK.
+
+Refer to the [MTK documentation](https://www.enterprisedb.com/docs/migration_toolkit/latest/)
+
+### Switch database connection to EDB Postgres
+
+Switching from Oracle is a simple step in VisionX. In the migrated VisionX application, simply click on the VisionX Settings screen and choose the “Datasources” tab. Click on “Edit” to change the database connection from Oracle to EDB Postgres Advanced Server.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration37.png">
+</p>
+
+## Create a VisionX application that uses an EDB Postgres Advanced Server / EDB Postgres Extended instance as a data source
+
+### Create a screen that connects to an existing table from an EDB Postgres Advanced Server / EDB Postgres Extended instance
+
+1. Data setup on Postgres Advanced Server 13 instance.
```

Comment:
For this example data was setup in an EDB Postgres Advanced Server 13 instance.
That is line 1.